src/NuGet.Clients/NuGet.VisualStudio.Client/NuGet.VisualStudio.Client.csproj

```diff
@@ -338,33 +338,14 @@
     <!-- dependency of Lucene.net. Can delete when Lucene.net has an update with a newer dependency on sharpziplib -->
   </ItemGroup>
   <ItemGroup>
-    <Content Include="eula.rtf">
+    <Content Include="extension.vsixlangpack">
+      <IncludeInVSIX>false</IncludeInVSIX>
+    </Content>
+    <Content Include="NOTICES.txt">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
As a best practice, `CopyToOutputDirectory` should be `PreserveNewest`.  Do you know why this is `Always`?  While you're here you might as well fix it unless there's some good reason that I'm unaware of.